### PR TITLE
Created login page

### PR DESCRIPTION
**Pull request by edenmaor**
Added a login page that extends the home page
Created a views.py inside side_hustle

please review code

Closed PR #31, #44
#41 Assigned by shay

### CODE REVIEW

**Review by assafg95 — APPROVED** [review]

**Review by AshkenYariv — APPROVED** [review]
@edenmaor Looks good! approved.

**Comment by assafg95** [comment]
Everything looks fine

File: templates/home.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    <div class="col-sm-4"> </div>
+<div class="col-md-4">
+
+<h1 class="text-center text-success"> Login page</h1>
+<br/>
+
+<div class="col-sm-12">
+
+  <ul class="nav nav-pills" >
+
```
Comment (shayakrach) [inline templates/home.html]: Why do you have too much line spacing here?

File: templates/home.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+</form>
+<br/>
+<a href="#" class="pull-right btn btn-block btn-success" > Already Register ?   </a>
+
+
+
+    </div>
+   </div>
+  </div>
+</div>
```
Comment (shayakrach) [inline templates/home.html:132]: I can't understand the indentation.

**Review by shayakrach — CHANGES_REQUESTED** [review]
Just fix the home.html page (I wrote some comments there)

File: templates/home.html [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+</div>
+</div>
```
Comment (shayakrach) [inline templates/home.html:95]: You should use indentation in your code for readability.

File: templates/home.html (hunk 1 above)
Comment (edenmaor) [reply templates/home.html]: easier to read the code like that

**Comment by shayakrach** [comment]
Also, fix the subject line.
You should capitalize the subject line.

File: templates/home.html (hunk 1 above)
Comment (yuvalk) [reply templates/home.html]: you should decide on coding style
usually extra newlines are discouraged

Comment (edenmaor) [reply templates/home.html]: @yuvalk You want me to change it?

**Review by assafg95 — APPROVED** [review]

**Review by shayakrach — APPROVED** [review]

**Comment by yuvalk** [comment]
I'm fine with this, but you'll have fix the conflicts

**Pull request merged**